components.json

```diff
@@ -1137,6 +1137,15 @@
 			"alias": "ts",
 			"owner": "vkbansal"
 		},
+		"typoscript": {
+			"title": "TypoScript",
+			"require": "javascript",
+			"alias": "tsconfig",
+			"aliasTitles": {
+				"tsconfig": "TSConfig"
+			},
```

Comment:
Given the popularity of TypeScript, I'm really not a fan of this alias. It's very confusing because if I hear "tsconfig", I'll think of [this](https://www.typescriptlang.org/docs/handbook/tsconfig-json.html).

Reply:
Well, you refer to a `json` file named `tsconfig`. But `tsconfig` is an own filetype. The syntax is the same as for `TypoScript`, but `TSConfig` controlls different parts of an installation. Those types, `TypoScript` and `TSConfig` are common. I would still suggest to add `TSConfig` as alias, because of the filetypes and the fact, that the label of Prism is correct when i highlight such code, because saying `TypoScript` there could be missleading where the settings should be located in the installation.

![grafik](https://user-images.githubusercontent.com/917005/89783899-0488e580-db18-11ea-814f-f1f40a2a0233.png)

Reply:
So `tsconfig` is a file extension. Then it's ok, I guess.